framework/ui/src/index.ts

```diff
@@ -90,6 +90,88 @@ export default {
 
       return true;
     });
+    defineRule(
```

Comment:
Definitely need to move rules definition to separate files, it's too much code here.